packages/shared/lib/core/wallet/utils/outputs/getMetadataFromNftOutput.ts

```diff
@@ -1,8 +1,10 @@
 import { FeatureType, MetadataFeature, NftOutput } from '@iota/sdk/out/types'
+import { DEFAULT_NFT_FEATURE_ENTRY_KEY } from '@core/nfts'
 
 export function getMetadataFromNftOutput(output: NftOutput): string {
     const metadata = output.immutableFeatures?.find(
         (feature) => feature.type === FeatureType.Metadata
     ) as MetadataFeature
-    return metadata?.data
+    // TODO: update this to return all entries
```

Comment:
same here, this can link to the gh issue so we know what to update later on and which issue to link